Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/Properties/launchSettings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "http://json.schemastore.org/launchsettings.json",
```

Comment:
Can't make a comment on the actual linke here but the applicationUrl should be set to port 35030